Gopkg.toml

```diff
@@ -53,6 +53,10 @@ required = [
   name = "k8s.io/code-generator"
   version = "kubernetes-1.12.6"
 
+[[override]]
+  name = "k8s.io/gengo"
+  revision = "bc9033e9ec9e0cec1552e9af7c079cb878dfc511"
```

Comment:
Since there is no tags in the repo can you comment like the constraints using revision above do? It helps clarify dependency age without checking out the repo.

Such as:
`# HEAD as of 2019-03-27`

Reply:
Done